examples/svm/plot_rbf_parameters.py

```diff
@@ -159,7 +159,7 @@ def __call__(self, value, clip=None):
 classifiers = []
 for C in C_2d_range:
     for gamma in gamma_2d_range:
-        clf = SVC(C=C, gamma=gamma)
+        clf = SVC(C=C, gamma=gamma, random_state=42)
```

Comment:
```suggestion
        clf = SVC(C=C, gamma=gamma)
```

As in the previous case, we don't use `probability=True` so this is already deterministic.